subprojects/language-native/src/integTest/groovy/org/gradle/language/cpp/CppApplicationPublishingIntegrationTest.groovy

```diff
@@ -422,42 +422,42 @@ class CppApplicationPublishingIntegrationTest extends AbstractInstalledToolChain
         main.parsedPom.scopes.isEmpty()
         def mainMetadata = main.parsedModuleMetadata
         mainMetadata.variants.size() == 6
-        mainMetadata.variant("debugWindowsRuntime").availableAt.coords == "some.group:test_debug_windows:1.2"
-        mainMetadata.variant("releaseWindowsRuntime").availableAt.coords == "some.group:test_release_windows:1.2"
-        mainMetadata.variant("debugLinuxRuntime").availableAt.coords == "some.group:test_debug_linux:1.2"
-        mainMetadata.variant("releaseLinuxRuntime").availableAt.coords == "some.group:test_release_linux:1.2"
-        mainMetadata.variant("debugMacosRuntime").availableAt.coords == "some.group:test_debug_macos:1.2"
-        mainMetadata.variant("releaseMacosRuntime").availableAt.coords == "some.group:test_release_macos:1.2"
-
-        def debug = repo.module('some.group', "test_debug_$currentOsFamilyName", '1.2')
+        mainMetadata.variant("debugWindowsX86Runtime").availableAt.coords == "some.group:test_debug_windows_x86:1.2"
```

Comment:
I agree with this for output locations, but the published metadata it seems like we should be as explicit as possible about what the file represents (at least for common things like os/architecture).  I could be persuaded otherwise, but I think it would be confusing to users to have some modules published one way and others published differently.

Reply:
I created https://github.com/gradle/gradle-native/issues/921 to tackle this as a follow up.